lib/src/pubspec.dart

```diff
@@ -381,8 +407,37 @@ class Pubspec extends PubspecBase {
           'The pubspec must be a YAML mapping.', pubspecNode.span);
     }
 
+    if (overridesContents != null) {
+      if (overridesLocation == null) {
+        throw ArgumentError('overridesLocation must be provided if '
+            'overridesContents is provided.');
+      }
+
+      YamlNode overridesNode;
+      try {
+        overridesNode =
+            loadYamlNode(overridesContents, sourceUrl: overridesLocation);
+      } on YamlException catch (error) {
+        throw PubspecException(error.message, error.span);
+      }
+
+      YamlMap overridesMap;
+      if (overridesNode is YamlScalar && overridesNode.value == null) {
+        overridesMap = YamlMap(sourceUrl: overridesLocation);
+      } else if (overridesNode is YamlMap) {
+        overridesMap = overridesNode;
+      } else {
+        throw PubspecException('The pubspec_overrides must be a YAML mapping.',
```

Comment:
What is `pubspec_overrides` there is `pubspec_overrides.yaml`

Reply:
I'm following the example a few lines above:
```The pubspec must be a YAML mapping.```

Should I change both?